src/components/NewsletterForm/index.tsx

```diff
@@ -71,7 +71,7 @@ export const NewsletterForm = ({
                             <input
                                 type="submit"
                                 className="bg-primary dark:bg-white text-white dark:text-black text-base shrink-0 grow-0 basis-auto font-bold lg:mt-0 border-none cursor-pointer px-5 py-3 md:py-2 rounded-full w-full md:w-auto"
-                                value="Subscribe"
+                                value="Try HogMail"
```

Comment:
I'm not crazy about this as a CTA.

1) 'Try' feels like the wrong verb. It's not like you get one issue as a sampler. You subscribe. 
2) HogMail is a term which isn't referenced in the other copy, may be unclear and isn't needed. 

IMO it's shorter and simpler just to say: `Sign me up!` or `Subscribe now!`

Reply:
Yeah I think 'Subscribe' or 'Sign me up!' is the sensible option, probably overthinking it a bit by trying to force the name in there. (I like 'Sign me up!' as it's more informal)

Reply:
Sign me up!